orchagent/neighorch.cpp

```diff
@@ -267,8 +290,8 @@ bool NeighOrch::addNextHop(const IpAddress &ipAddress, const string &alias)
     {
         if (setNextHopFlag(nexthop, NHFLAGS_IFDOWN) == false)
         {
-            SWSS_LOG_WARN("Failed to set NHFLAGS_IFDOWN on nexthop %s for interface %s",
-                ipAddress.to_string().c_str(), alias.c_str());
+            SWSS_LOG_WARN("Failed to set NHFLAGS_IFDOWN on nexthop %s",
```

Comment:
Revert back to original

Reply:
It is not appropriate to revert this log to previous, since it will not contain full NH description.

Reply:
@prsunny New NextHopKey::toLogString() function maintains the original output.

Reply:
@prsunny All logs are reverted to previous format.